src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/PersonOfSignificantControl.java

```diff
@@ -12,14 +12,15 @@
     private String appointmentDate;
     private String superSecurePscInd;
     private String officerNationality;
+    private String usualResidentialCountry;
     private String officerDateOfBirth;
     private String suppliedCompanyName;
     private String natureOfControl;
     private String registerLocation;
     private String registrationNumber;
     private String lawGoverned;
     private String legalForm;
-    private String countryOfResidence;
+    private String pscCountry;
```

Comment:
what does the pscCountry represent? is it the same as the country of residence?

Reply:
I believe so but psc country comes from a different table CORPORATE_APPT_DETAIL, which is not where we want to get this data from which is OFFICER_DETAIL.USUAL_RESIDENTIAL_COUNTRY.

The psc_country field was added as contry of residence because at the time it was beleived that this was the only source of that information but as this is not the case it is better to remove dit from the oracle query to avoid confusion and then make it clear in the cs api where we get the country of residence information from.

The psc_country field as added as part of the rle data.

Reply:
ok, we can look at removing it from the query if we don't need it.